app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/SessionData.scala

```diff
@@ -20,17 +20,15 @@ import play.api.libs.json.{Format, Json}
 
 final case class SessionData(
   ivContinueUrl: Option[String],
-  businessPartnerRecord: Option[BusinessPartnerRecord],
+  subscriptionStatus: Option[SubscriptionStatus],
```

Comment:
this is one the biggest things that has changed here - I've made it so that we store the status of the journey with the necessary information instead of separate bits of information floating around to make it easier to see where the user is on their journey